server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
N.B.: "current" to "current" rolling upgrade is already tested (and runs by default) in `ExampleConfigsIT#testHotRodRollingUpgrades`, resp. `ExampleConfigsIT#testRestRollingUpgrades`